### ISSUE TITLE
RoutingError after destroying and recreating

### ISSUE THREAD

**Issue by eugenk** [ontohub/ontohub#1630]
When I create a Repository with name "demo"
And I upload [periods.dol](https://ontohub.org/repositories/colore-sample/d290f729ab4a630d212a9114c88fdbf24ea689d3/show/periods/mappings/periods.dol)
And I attempt to prove every Theorem in its child ontology "i2__T"
And I destroy the repository via `Repository.find_by_path('demo').destroy`
And I create a Repository with name "demo"
And I upload [periods.dol](https://ontohub.org/repositories/colore-sample/d290f729ab4a630d212a9114c88fdbf24ea689d3/show/periods/mappings/periods.dol)
And I attempt to prove every Theorem in its child ontology "i2__T"
And I visit http://localhost:3000/demo/periods//i2__T//Ax_0_1//proof-attempt-1
Then I get a RoutingError:
```
No route matches [GET] "/demo/periods/i2__T/Ax_0_1/proof-attempt-1"
```

**Comment by tillmo** [comment]
idea: dump the database after the first and after the second proof attempt, and compare the dumps with kdiff3.

**Comment by eugenk** [comment]
This could have been solved with #1618.
